index.js

```diff
@@ -313,11 +338,16 @@ instance.prototype.action = function(action) {
 			break;
 
 		case "release_cl":
-			cmd = "RQL " + opt.cuelist;
+			self.system.emit('variable_parse', opt.cuelist, function (value) {
+				eval_1 = value
+			})
+			cmd = "RQL " + eval_1;
 			break;
 	}
 
 	if (cmd !== undefined) {
+		console.log('command')
+		console.log(cmd)
```

Comment:
This would probably be better:
```suggestion
		debug('command ' + cmd)
```